backend/src/main/java/com/bakdata/conquery/sql/execution/CDateSetParser.java

```diff
@@ -0,0 +1,9 @@
+package com.bakdata.conquery.sql.execution;
+
+import com.bakdata.conquery.models.common.CDateSet;
+
+public interface CDateSetParser {
+
+	CDateSet fromString(String multiDateRange);
```

Comment:
Der Parser liest unsere Interval-Notation Darstellung ein und wandelt den Multi-Daterange String in ein CDateSet um. Außerdem wird -1 Tag vom exkludierten Enddatum abgezogen, also das Enddatum inkludiert.

Damit erreichen wir dann die erwünschte Darstellung von validity dates.